fmriprep/cli/parser.py

```diff
@@ -379,8 +378,7 @@ def _fallback_trt(value, parser):
 Important to note, the ``res-*`` modifier does not define the resolution used for \
 the spatial normalization. To generate no BOLD outputs, use this option without specifying \
 any spatial references. For further details, please check out \
-https://fmriprep.readthedocs.io/en/%s/spaces.html"""
-        % (currentv.base_version if is_release else 'latest'),
+https://fmriprep.readthedocs.io/en/{currentv.base_version if is_release else 'latest'}/spaces.html""",
```

Comment:
The string you're editing is not an f-string.

Reply:
Fixed. Force pushed in 9a520e9.